miniupnpd/genconfig.sh

```diff
@@ -233,6 +233,7 @@ case $OS_NAME in
 			FW=pf
 		fi
 		echo "#define USE_IFACEWATCHER 1" >> ${CONFIGFILE}
+		echo "#define PFRULE_INOUT_COUNTS" >> ${CONFIGFILE}
```

Comment:
shouldn't the DragonFly version be tested ?
was done for OpenBSD : 
https://github.com/miniupnp/miniupnp/blob/master/miniupnpd/genconfig.sh#L144